File: Multiples of 3 and 5/mutiplesof.js

function mutiplesof(a,b,to) {
    var list = []
    var result = null
    while (--to) {
            if (to == 0) break
            if (  (to % a) == 0 ) list.push(to)
            if ((to % b )== 0) list.push(to)
        }
    result = list.reduce((a,b)=>a+b)
    return result
    }

var threeand5 = mutiplesof(3,5,1e3)
console.log(threeand5) 

File: Multiples of 3 and 5/mutiplesof.py
from functools import reduce


def multiplesof(a,b,to):
    # Mylsit is the list
    # result is the final value
    mylist,result = [] , None
    result = 0
    while to  != 0:
        to -= 1
        if ((to % a) == 0) :mylist.append(to)
        if ((to % b) ==0 ) : mylist.append(to)
    result = reduce(lambda a,b: a+b,mylist)
    return result
print(multiplesof(3,5,1000))
